### DISCUSSION TITLE
Complete Noob - How and where do I start

### DISCUSSION THREAD

**Discussion by HedgePiglet** [icloud-photos-downloader/icloud_photos_downloader#707] [Q&A]
Hi forum,
This looks like a great tool. But I have no idea where to start and I was hoping someone could help me with a short step by step. Win10(64) and I have downloaded the executable, run it and it asks for my email address which I input and then it closes.

I am clearly missing vital steps if someone can assist? Thank you.

**Comment by AndreyNikiforov** [comment]
see Windows-related [comment](https://github.com/icloud-photos-downloader/icloud_photos_downloader/issues/686#issuecomment-1728124755)

It may worth updating doc for Windows users as command line interface (CLI) is not native on that platform for most users.
